# This message expresses possible proximity pairs.

Header header

# Proximity pairs with their indexes. These are not links but predefined collision objects in the robot configuration header.
int32[] int_pair_a
int32[] int_pair_b

# Proximity pairs with their indexes. These are not links but predefined collision objects in the robot configuration header.
int32[] ext_pair_robot
int32[] ext_pair_object